src/Components/Table/Rows.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import FakeBookings from "../../data/fakeBookings.json";
+import moment from "moment";
+
+const Rows = () => {
```

Comment:
Don't you think we should receive this `FakeBookings` via props rather than import it again in here? 

You are using this component in SearchResults.js, where we could import this data and then provide it to the components via props.

Reply:
Oh, yes, I did PROPS for data and I use it in columns component and I IMPORT it in rows component to see if it makes any difference . THANK YOU